Assets/Project/Scripts/Modules/MenuSelectScene/LevelSelect/LevelSelectDirector.cs

```diff
@@ -11,19 +13,18 @@ public class LevelSelectDirector : SingletonObject<LevelSelectDirector>
         /// <summary>
         /// 木
         /// </summary>
-        private static List<LevelTreeController> _trees;
+        private List<LevelTreeController> _trees;
 
         /// <summary>
         /// 道
         /// </summary>
-        private static List<RoadController> _roads;
+        private List<RoadController> _roads;
 
         private void Awake()
         {
-            _trees = GameObject.FindGameObjectsWithTag(Constants.TagName.TREE)
-                .Select(tree => tree.GetComponent<LevelTreeController>()).ToList<LevelTreeController>();
-            _roads = GameObject.FindGameObjectsWithTag(Constants.TagName.ROAD)
-                .Select(road => road.GetComponent<RoadController>()).ToList<RoadController>();
+            _trees = GameObject.FindGameObjectsWithTag(Constants.TagName.TREE).Select(tree => tree.GetComponent<LevelTreeController>()).ToList();
+            _roads = GameObject.FindGameObjectsWithTag(Constants.TagName.ROAD).Select(road => road.GetComponent<RoadController>()).ToList();
+            FindObjectOfType<ResetController>(true).OnDataReset.Subscribe(_ => Reset()).AddTo(this);
```

Comment:
```suggestion
            FindObjectOfType<ResetController>(true).OnDataReset
                .Subscribe(_ => {
                    _trees.ForEach(tree => tree.Reset());
                    _roads.ForEach(road => road.Reset());
                })
                .AddTo(this);
```
にして `private void Rest()` を消そう

Reply:
6744aa0